### DISCUSSION TITLE
Prevent automatic downloading of Firefox for every ML4W update

### DISCUSSION THREAD

**Discussion by tko76** [mylinuxforwork/dotfiles#459] [Q&A]
Hi ML4W Experts,

My default browser is Zen, every ML4W update causes Firefox to download and install which I then have to delete post update. Is there any way to prevent this from downloading as the default browser for every update released?

Second similar question, why does the Papirus icon set always re-download even though there are no changes?

extra/firefox                          132.0.2-1    243.27 MiB      70.96 MiB
extra/papirus-icon-theme  20240501-1   20240501-1     0.00 MiB               

Thanks in advance!

**Comment by Veicm** [comment]
The first one: after you done a update you don't have to delete the Firefox app because you can simply open the ml4w dotfiles settings app and select the default browser you want. This setting should still be remembered by the program after an update.

And for the second one: I have no idea. I'm also annoyed by this.
